fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java

```diff
@@ -518,6 +518,7 @@ public HdfsFs getDistributedFileSystem(String scheme, String path, Map<String, S
             }
             return fileSystem;
         } catch (Exception e) {
+            Thread.interrupted();
```

Comment:
Is this the right way to handle the interuption event? Do we need to do something like retry? Always clear the interupted state even though there's no interuption happened? If we really need to eat the interupte exception, we should catch this type of exception explicitly and comment the reason i think.

Reply:
If a exception is caught, the interrupted flag set by the exception should usually be reset to not affect the caller thread, otherwise it is meaningless to catch this exception, just rethrow it and let the caller to handle it will be better.
the retry is done in the underlying code, no need to retry here.
if let the interrupted flag set, any subsequent blocking code in this thread will throw exception, like `sleep`, `wait`, `read`, `write` ...,  it's impossible to catch all the exceptions in all blocking code.